### DISCUSSION TITLE
Show OTP provider as name in MFA

### DISCUSSION THREAD

**Discussion by sebastianbuechler** [zitadel/zitadel#4169] [Ideas]
Currently in zitadel.cloud there's an overview of all registered MFA factors. U2F factors have the option to be named by the user, but OTP factors do not have this possibility. Maybe it would make sense to display the OTP provider name (like Google Authenticator) in order to help a user remind which OTP provider he uses. Or is it not clear which one is in registerred?

![image](https://user-images.githubusercontent.com/42775578/184317957-5f0be495-271b-460d-81ec-fbadeab10bde.png)

**Comment by hifabienne** [comment]
Hi @sebastianbuechler 
Thanks for your idea.
Actually, we don't know which provider the user registered.
What would be possible to allow the user to enter the name itself.
Would that solve your problem? And would you enter the name if you have to do it manually?

**Reply by sebastianbuechler** [reply]
Yes, that would solve it as well. I guess I would do it in order to remember which provider I chose, but can't speak for others.
